cmd/crossplane/main.go

```diff
@@ -37,48 +36,57 @@ import (
 	"github.com/crossplane/crossplane/cmd/crossplane/rbac"
 )
 
+type debugFlag bool
```

Comment:
It's perhaps not so bad in this case but we typically try to avoid package scoped variables. Is there some way to avoid using one here?

Reply:
Hmm, the preferred way to add hooks such as this one with Kong is generally using such package scoped variables. I am not sure of any workaround for defining hooks.